client/src/main/java/io/pravega/client/stream/RetentionPolicy.java

```diff
@@ -33,18 +32,13 @@
         /**
          * Set retention based on the total size of the data in the stream in bytes.
          */
-        SIZE,
-
-        /**
-         * Set retention policy based on Consumption.
-         * Also see: {@link ReaderGroupConfig.StreamDataRetention}
-         */
-        CONSUMPTION
+        SIZE
     }
 
     private final RetentionType retentionType;
-    private final long retentionParam;
-    private final ConsumptionLimits consumptionLimits;
+    private final long retentionMin;
```

Comment:
please dont change the name of the variable otherwise anyone using retention policy in their application would have to recompile their code.

Reply:
in fact to ensure that this change is backward compatible and we dont break the application, perhaps we can simply have a getter with the original name which returns the min.

Reply:
Fixed.